Boundaries-components.md

```diff
@@ -0,0 +1,67 @@
+
+### Boundaries, components, and user interactions
+Every data.gov component follows the pattern described below with variance only in the application or data services in use unless otherwise specified.
+
```

Comment:
Can you add the preview image here?

Reply:
![image](https://user-images.githubusercontent.com/75858167/103218301-a141f900-48e8-11eb-8423-5ebabdfb26d0.png)